app/javascript/src/Policies/middleware/PolicyChain.jsx

```diff
@@ -16,7 +16,7 @@ function convertToChainPolicy (registryPolicy: RegistryPolicy, storedPolicy: Sto
     ...registryPolicy,
     ...{
       enabled: storedPolicy.enabled,
-      configuration: storedPolicy.configuration,
+      data: storedPolicy.configuration,
```

Comment:
For consistency, I think it should be either `data: storedPolicy.data` or `configuration: storedPolicy.configuration`.

Reply:
The thing is what APIcast stores it as `configuration` we used it as the data for the form schema and use `configuration` for the schema itself. I think it will make more sense with the new structure to call it `configuration` since we are going to be storing the custom ones as `schema`

Reply:
Still, this will be attended in a following PR